### ISSUE TITLE
Selection using drag.

### ISSUE THREAD

**Issue by GaurangGhinaiya** [angular-ui/ng-grid#3968]
Hello,

I want to select multiple row using mouse drag.

Is it possible?

Any help greatly appreciated.

**Comment by PaulL1** [comment]
No, this is not currently possible.  You can, however, use shift click.
